src/main/java/bio/terra/cli/command/workspace/Clone.java

```diff
@@ -0,0 +1,81 @@
+package bio.terra.cli.command.workspace;
+
+import bio.terra.cli.businessobject.Context;
+import bio.terra.cli.businessobject.Resource;
+import bio.terra.cli.businessobject.Workspace;
+import bio.terra.cli.command.shared.BaseCommand;
+import bio.terra.cli.command.shared.options.Format;
+import bio.terra.cli.command.shared.options.WorkspaceNameAndDescription;
+import bio.terra.cli.serialization.userfacing.UFClonedResource;
+import bio.terra.cli.serialization.userfacing.UFClonedWorkspace;
+import bio.terra.cli.serialization.userfacing.UFWorkspace;
+import bio.terra.workspace.model.CloneResourceResult;
+import bio.terra.workspace.model.ClonedWorkspace;
+import bio.terra.workspace.model.ResourceCloneDetails;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import picocli.CommandLine;
+import picocli.CommandLine.Command;
+
+@Command(name = "clone", description = "Clone an existing workspace.")
+public class Clone extends BaseCommand {
+
+  @CommandLine.Option(
+      names = "--location",
+      required = false,
+      description = "Location for newly created resources.")
+  private String location;
+
+  @CommandLine.Mixin private WorkspaceNameAndDescription workspaceNameAndDescription;
+
+  @CommandLine.Mixin private Format formatOption;
```

Comment:
Let's also include the `WorkspaceOverride` mixin + method call here, since we provide it for all other commands that require a workspace.